test/json/image_stream.json

```diff
@@ -0,0 +1,21 @@
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "eng-boss-image-inspector",
+    "namespace": "default",
+    "selfLink": "/oapi/v1/namespaces/default/imagestreams/eng-boss-image-inspector",
+    "uid": "f4499f16-b7bc-11e5-a0a4-001a4a231290",
+    "resourceVersion": "2836367",
+    "creationTimestamp": "2016-01-10T17:09:48Z",
+    "annotations": {
+      "openshift.io/image.dockerRepositoryCheck": "the registry \"https://rcm-img-docker01.build.eng.bos.redhat.com:5001\" could not be reached"
```

Comment:
kinda weird annotation, isn't it? @smarterclayton, any idea why this is an annotation and not a status/err message?